src/main/webapp/app/exercises/shared/result/result.component.ts

```diff
@@ -292,6 +292,13 @@ export class ResultComponent implements OnInit, OnChanges {
         return false;
     }
 
+    /**
+     * Checks whether the participation is a student programming participation
+     */
+    isStudentProgrammingParticipation() {
```

Comment:
I'm not sure if this is also affected by what @krusche told us today in the meeting with the problematic getters, could you maybe check that?